test/test_shared.cpp

```diff
@@ -358,6 +358,37 @@ TEST(Shared_CompactingOnTheFly)
     }
 }
 
+TEST(Shared_EncryptedRemap)
+{
+    // Attempts to trigger code coverage in util::mremap() for the case where the file is encrypted.
+    // This requires a database size which is non-divisible by page_size() *and* is bigger than
```

Comment:
How can this be true? AFAIK we always have database sizes which are divisible by page_size()... perhaps except for those written by compact()